CHANGELOG.md

```diff
@@ -33,6 +33,12 @@
   ```
   
   In Dart SDK `>=1.25.0` this can be relaxed as `part of` can refer to a path.
+  To opt-in, `GeneratorBuilder` now has a new flag, `requireLibraryDirective`.
```

Comment:
Sounds good - if it hasn't landed officially in any release there is no sdk constraint we can set either right.... so we can't really release this feature at all until then.